code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1141,37 +1141,39 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 		if(module.type == /obj/item/robot_module/janitor)
 			var/turf/tile = loc
 			if(isturf(tile))
```

Comment:
up here; make it first, as well, so we short circuit the `isturf` check when it's dead.